pytest.ini

```diff
@@ -1,14 +1,14 @@
 [pytest]
 addopts =
     # `pytest-xdist`:
-    --numprocesses=auto
+    # --numprocesses=auto
```

Comment:
You could override this all in `tox.ini` or use `tox -- -n 0 --testmon-off --durations 0` and so on

Reply:
I keep forgetting to remove that from the changelist.

I have to disable those for my debugging workflow. I can't seem to get tox to operate correctly when working with the PyCharm debugger. I was, however, able to get pytest to be stable when modifying the pytest configuration in this manner.

I'm not sure of a better way to go about it, any suggestions?

Reply:
Oh, in this case, it's probably fine. I haven't used PyCharm in ages — have you tried their remote debugger mode?

Reply:
No, but I believe the effect would be the same. Whatever debugger PyCharm uses internally disallows the usage of one of the packages loaded by `testmon`, and `xdist` causes issues where breakpoints may or may not trigger. There also appears to be an issue where if `testmon` is left loaded it sometimes fails to detect any tests at all.

I'll make sure to revert this change and update :)

Reply:
Right, testmon is a _very_ tricky tool intended to be used mostly during the development.